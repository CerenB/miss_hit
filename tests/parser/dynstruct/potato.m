function [x, y] = potato()
  x = 1;
  y = 2;
end
