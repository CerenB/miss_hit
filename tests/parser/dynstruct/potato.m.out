----------------------------------------------------------------------
--  Parse tree for potato.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <potato>
      Outputs: Identifier <x>
      Outputs: Identifier <y>
    Body: Sequence_Of_Statements
      Statements: Simple_Assignment_Statement
        LHS: Identifier <x>
        RHS: Number_Literal <1>
      Statements: Simple_Assignment_Statement
        LHS: Identifier <y>
        RHS: Number_Literal <2>
----------------------------------------------------------------------
In potato.m, line 1
| function [x, y] = potato()
|          ^ info: this token is not linked to the ast
In potato.m, line 1
| function [x, y] = potato()
|            ^ info: this token is not linked to the ast
In potato.m, line 1
| function [x, y] = potato()
|               ^ info: this token is not linked to the ast
In potato.m, line 1
| function [x, y] = potato()
|                 ^ info: this token is not linked to the ast
In potato.m, line 1
| function [x, y] = potato()
|                         ^ info: this token is not linked to the ast
In potato.m, line 1
| function [x, y] = potato()
|                          ^ info: this token is not linked to the ast
In potato.m, line 2
|   x = 1;
|        ^ info: this token is not linked to the ast
In potato.m, line 3
|   y = 2;
|        ^ info: this token is not linked to the ast
In potato.m, line 4
| end
| ^^^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
